docs/snippets/snippets.toml

```diff
@@ -13,6 +13,7 @@ log_line = [ # Not a complete example -- just a single log line
   "rust",
   "py",
 ]
+spatial3dview = ["cpp", "rust"] # Missing examples
```

Comment:
think we should put all space view snippets into their own folder. That way we can easily add checks like "Make sure all spaceviews has a snippet"

Reply:
https://github.com/rerun-io/rerun/issues/6133